### DISCUSSION TITLE
Open Tasks plugin modal dialog directly from card-board

### DISCUSSION THREAD

**Discussion by ItsFullOfCode** [roovo/obsidian-card-board#104] [Ideas]
- If Tasks Plugin is installed (or if the user activates a card-board setting): let the user open the Tasks plugin’s modal editor from within the board. An icon button within the task's card in card board would open the Tasks' plugin dialog and allow the user to make the desired changes without leaving cardboard.

**Comment by roovo** [comment]
Thanks - I am discussing how this (and some other things) could work with the Tasks plugin with [Clare Macrae](https://github.com/claremacrae).  Not sure on the outcome or timescales yet, but will update here as and when :)
